content/for-developers/tutorials/inner-loop/05-expose-app/expose-app.md

```diff
@@ -1,100 +1,147 @@
+    ![svc-values](images/svc-values.png)
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: nordmart-service
-spec:
-  type: NodePort
-  ports:
-    - port: 80
-      targetPort: 8080
-  selector:
-    app: nordmart
-```
+    > You n change or add any configuration for the service. To see more configurations [click](https://github.com/stakater/applition.git).
 
-In the above example, a service named `nordmart-service` is defined with the NodePort type, exposing port 80 and forwarding traffic to port 8080 of the application.
+1. Run `tilt up` at the root of your directory. Hit the space bar and the browser with `TILT` logs will be shown. If everything is green then the changes will be deployed on the cluster.
 
-### NodePort
+1. Let's go to the services under the networking section in your namespace
 
-When using the NodePort service type, OpenShift assigns a random port within a predefined range on each node of the cluster. To access your application externally, you need to determine the assigned port and the IP address of any of the cluster's nodes:
+    The service has exposed our review pod on port `8080`.
 
-a. Retrieve the port assigned to the service by running the following command:
+    ![find service](images/svc.png)
 
-`oc get service nordmart-service`
+## Tutorial #2
 
-b. Identify the port assigned to the NodePort field. This port will be used to access your application externally.
+### Exposing Your Application to External Traffic via Route
 
-c. Obtain the IP address of any node in the cluster:
+OpenShift provides a routing mechanism called "routes" that allows you to expose applications using hostnames and paths. Routes are created using the Route resource and can provide additional features such as SSL termination and path-based routing.
 
-`oc get nodes -o wide`
+To create a route:
 
-d. Combine the node's IP address with the assigned port to access your application. For example: `http://<node-ip>:<assigned-port>`
+1. In your `deploy/values.yaml` file, define a route:
 
-### LoadBalancer
+    ```yaml
+    ## Route
+    route:
+      enabled: true # Enables the creation of a Route resource on SAAP
+      port:
+        targetPort: http  # Specifies the target service port name
+    ```
 
-If your OpenShift cluster is running in a cloud environment that supports load balancers, you can use the LoadBalancer service type. This type automatically provisions a load balancer to distribute traffic to your application:
+    It should look like this:
 
-a. In your deployment configuration YAML file, define a service with the type LoadBalancer:
+    ![route-values](images/route-values.png)
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: nordmart-service
-spec:
-  type: LoadBalancer
-  ports:
-    - port: 80
-      targetPort: 8080
-  selector:
-    app: nordmart
-```
+    `targetPort: http` represents the port on which the backend service is listening to handle incoming traffic.
```

Comment:
you can remove this since you ve specified it in line 62

Reply:
Done